gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -785,8 +733,8 @@ def delete_device(self, event):
                                event_data=device)
 
     @poll_event_desc(event='DEVICE_BEING_DELETED', spacing=2)
-    @nfp_log.patch_method
     def check_device_deleted(self, event):
+        nfp_logging.store_logging_context(**event.context)
```

Comment:
whereever nfp_id is first time received and also in event handlers - handle_event and handle_poll_event and poll_event_desc methods